openapi/openapi.json

```diff
@@ -73,7 +77,11 @@
                                     "example": {
                                         "data": {
                                             "attributes": {
-                                                "templateName": "this-does-not-exist"
+                                                "templateName": "this-does-not-exist",
+                                                "owner": {
+                                                    "id": "urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6",
```

Comment:
https://www.rfc-editor.org/rfc/rfc2141#section-2

instead of `uuid` as the namespace (it is more of a format in my eyes), could we have an identifier for our service/issuer. Something like:

`urn:cica:56P4CMsGh_02YOlWpd8PAOI-2sVlB2nsNU7mcLZYhYw=`

perhaps we should follow [One Logins example](https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/process-identity-information/#process-your-user-s-identity-information) and use `fdc` described [here](https://www.iana.org/assignments/urn-namespaces/urn-namespaces.xhtml)